tools/gulp/task_helpers.ts

```diff
@@ -207,3 +207,41 @@ export function openFirebaseDatabase() {
 export function isTravisPushBuild() {
   return process.env['TRAVIS_PULL_REQUEST'] === 'false';
 }
+
+/** Decode the token for Travis to use. */
+function decode(str: string): string {
+  return (str || '').split('\\n').reverse().join('\\n').replace(/\\n/g, '\n');
+}
```

Comment:
A bit nitpicky, can we move that function below the `openXXX()` functions? (so we have it grouped)